### PR TITLE
Add Core manifest

### PR DESCRIPTION
**Pull request by JohnCorevolta**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66642)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Singleton is discontinued; please move to multi-file version 1.1.0

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: ManticoreGames.Core
+PackageName: Core
```
Comment (ItzLevvie) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml:2]:
```suggestion
```
This isn't needed here as it's meant to be in the locale file.

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 1.3.0.0
+PackageLocale: en-US
+Publisher: Manticore Games
+License: Copyright © 2020 Manticore Games, Inc. All Rights Reserved.
+ShortDescription: Core is an endless universe of games and worlds designed by a global community of creators.
```
Comment (ItzLevvie) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml:7]:
```suggestion
```
This isn't needed here as it's meant to be in the locale file.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by JohnCorevolta** [comment]
Looks like the docs at https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md are wrong, all the `URL` fields should be `Url`?

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
+InstallerType: BURN
```
Comment (Trenly) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml:3]:
```suggestion
InstallerType: burn
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
+PackageLocale: en-US
+Publisher: Manticore Games
+PublisherUrl: https://www.coregames.com
+PrivacyUrl: https://www.coregames.com/legal/ManticorePrivacyPolicy.html
+PackageName: Core
+PackageUrl: https://www.coregames.com
+License: Copyright © 2020 Manticore Games, Inc. All Rights Reserved.
+LicenseUrl: https://www.coregames.com/legal/ManticoreLegalNotices.html
```
Comment (Trenly) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.locale.en-US.yaml:10]: A copyright is not a license; Please update this
```suggestion
License: Proprietary
LicenseUrl: https://www.coregames.com/legal/ManticoreLegalNotices.html
Copyright: Copyright © 2020 Manticore Games, Inc. All Rights Reserved.
```

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml (hunk 3 above)
Comment (JohnCorevolta) [reply manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml]: Should probably be updated here then https://docs.microsoft.com/en-us/windows/package-manager/winget/#supported-installer-formats

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
```
Comment (ItzLevvie) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.installer.yaml:2]:
```suggestion
PackageVersion: 1.3.1.0
```
This must match Control Panel.

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
```
Comment (ItzLevvie) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.locale.en-US.yaml:2]:
```suggestion
PackageVersion: 1.3.1.0
```
This must match Control Panel.

File: manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.yaml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
```
Comment (ItzLevvie) [inline manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.yaml:2]:
```suggestion
PackageVersion: 1.3.1.0
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also need to update the package version folder from `manifests/m/ManticoreGames/Core/1.3.0.0` to `manifests/m/ManticoreGames/Core/1.3.1.0`

**Review by ItzLevvie — APPROVED** [review]

**Comment by JohnCorevolta** [comment]
@ItzLevvie Thank you for your patience! 🙏

**Pull request merged**